src/odemis/dataio/tiff.py

```diff
@@ -1705,6 +1705,10 @@ def read_data(filename):
     raises:
         IOError in case the file format is not as expected.
     """
+    # TODO: support filename to be a File or Stream (but it seems very difficult
```

Comment:
No need to put _ensure_fs_encoding() back here in read_*(), it's now in open_data().